kentikapi/client.go

```diff
@@ -177,7 +179,12 @@ func makeConnForSynthetics(c Config) grpc.ClientConnInterface {
 	if c.SyntheticsGRPCAPIURL == "" {
 		c.SyntheticsGRPCAPIURL = syntheticsGRPCAPIURL
 		config := &tls.Config{}
-		conn, err := grpc.Dial(syntheticsGRPCAPIURL, grpc.WithTransportCredentials(credentials.NewTLS(config)))
+		unaryInterceptor := c.createUnaryInterceptor()
+		conn, err := grpc.Dial(
+			syntheticsGRPCAPIURL,
+			grpc.WithTransportCredentials(credentials.NewTLS(config)),
+			grpc.WithUnaryInterceptor(unaryInterceptor),
```

Comment:
Instead of assigning the interceptor function to `unaryInterceptor` variable I can do it inline.